docs/sources/datasources/elasticsearch/_index.md

```diff
@@ -140,6 +140,9 @@ For more information about AWS authentication options, refer to [AWS authenticat
 You can define and configure the data source in YAML files as part of Grafana's provisioning system.
 For more information about provisioning, and for available configuration options, refer to [Provisioning Grafana]({{< relref "../../administration/provisioning/#data-sources" >}}).
 
+> **Note:** `database` [field is deprecated](https://github.com/grafana/grafana/pull/58647).
+> We suggest to use `index` field in `jsonData`. Please see the examples below.
+
```

Comment:
i'd recommend one of two things:
1. do not mention it here at all
2. rewrite it a little, because hmm.. we do not mention `database` anywhere else in the text, so the user may be confused why we are talking about it. so maybe something like: "previously we used the `database` field to store the index name. this is now deprecated and the  `index` field in `jsonData` should be used" (but with better english 😁 )